keydates-worker/keydates_worker.py

```
@@ -488,6 +498,82 @@ def appget(method, params):
+
+def venue_timezone(events, tzmap):
+    """Timezone for a con's posts: the zone of its soonest upcoming edition that
+    has one (editions of one con virtually never straddle zones). UTC when none."""
+    import zoneinfo
+    for e in sorted(events, key=lambda e: e.get("startDate", "")):
+        tz = tzmap.get(e["id"])
+        if not tz:
+            continue
+        try:
+            zoneinfo.ZoneInfo(tz)
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)